sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient.cs

```diff
@@ -0,0 +1,969 @@
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Close the client connection.
+        /// </summary>
+        /// <param name="connectionId">Target connection Id.</param>
+        /// <param name="reason">The reason closing the client connection.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        public virtual async Task<Response> CloseClientConnectionAsync(string connectionId, string reason = default, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.CloseClientConnectionAsync");
+            scope.Start();
+            try
+            {
+                var response = await RestClient.CloseClientConnectionAsync(hub, connectionId, reason, cancellationToken).ConfigureAwait(false);
+                return response;
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Close the client connection.
+        /// </summary>
+        /// <param name="connectionId">Target connection Id.</param>
+        /// <param name="reason">The reason closing the client connection.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response CloseClientConnection(string connectionId, string reason = default, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.CloseClientConnection");
+            scope.Start();
+            try
+            {
+                var response = RestClient.CloseClientConnection(hub, connectionId, reason, cancellationToken);
+                return response;
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Remove a connection from the target group.
+        /// </summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="connectionId">Target connection Id.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        public virtual async Task<Response> RemoveConnectionFromGroupAsync(string group, string connectionId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.CloseClientConnectionAsync");
```

Comment:
(Just in case it's not clear, we need to patch up every one of these.  All your unit tests will fail because of them.)

Reply:
done